### ISSUE TITLE
How to navigate using keyboard in "Exit JWM" confirmation dialog?

### ISSUE THREAD

**Issue by limelime** [joewing/jwm#190]
How to navigate using keyboard in "Exit JWM" confirmation dialog?
Using keyboard, how to choose between "OK" and "Cancel"?

**Comment by joewing** [comment]
By default, pressing return selects "OK" and pressing escape selects "Cancel".  You can change that key binding, of course (the "select" and "escape" options).

**Comment by technosaurus** [comment]
I think what limelime was looking for is "tab-ability".  IIRC, there is no indicator which button is currently "active" so that pressing enter selects that button.  I could see someone pressing tab, enter and expecting the dialog to cancel.  This is something that could be revisited if you decide to add accessibility features to jwm... for instance passing text, etc... from the currently active item to a user defined program (for example an espeak wrapper for vision impaired users)

**Comment by limelime** [comment]
+1 if considering accessibility feature as suggested by technosaurus.

**Comment by limelime** [comment]
> By default, pressing return selects "OK" and pressing escape selects "Cancel". You can change that key binding, of course (the "select" and "escape" options).

On my Debian 8, I'm using JWM vsvn-579 and return or escape key don't react on the Exit dialog.

The jwmrc file contains the followings:
```xml
...
  <Key key="Return">select</Key>
  <Key key="Escape">escape</Key>
...
```

```
~>xmodmap -pke | grep -iE 'escape|return'
keycode   9 = Escape NoSymbol Escape
keycode  36 = Return NoSymbol Return

```

There is no error on `jwm -p`.

**Comment by joewing** [comment]
579 is quite old, (574 commits ago, January 2012!).  So I really have no idea if this worked back then.

**Comment by joewing** [comment]
I agree with @technosaurus though, there should be a way to tab between OK and cancel.
